lxdock/container.py

```diff
@@ -246,6 +272,11 @@ def status(self):
                 constants.CONTAINER_STOPPED: 'stopped',
             }.get(container.status_code, default_status)
         return status
+    
+    @property
+    def get_lxd_host(self):
+        """ Returns a string identifier representing the current host of the container. """
+        return get_container_host(self.lxd_name)
```

Comment:
It could be good to cache the host once you've found it since it's unlikely to move by itself.

Reply:
yeah, I was thinking to do this too... also, because there could be a race condition if a container is created and service discovery doesn't update fast enough.